spec/rails_helper.rb

```diff
@@ -118,6 +118,27 @@
     Course::IDENTIFIERS.each do |identifier|
       FactoryBot.create(identifier)
     end
+    # create cohorts since 2021 with default schedule
```

Comment:
I think I would rather avoid this for now since some tests are creating specific cohorts and schedules, we can revisit this in a later ticket if that's ok?